docs/general/community-standards/servers.md

```diff
@@ -0,0 +1,26 @@
+---
+uid: community-standards-servers
+title: Server Policy
+---
+
+# Server Policy
+
+Jellyfin is a free and open source project that provides the **tools to run your own server** for hosting your media, and clients to connect to one of those servers.
+
+The Jellyfin Project itself, and members of the Jellyfin team, **do not provide servers for you to use**, with the sole exception of [our demo server for evaluation and testing](https://demo.jellyfin.org/stable).
+
+**If you are using or paying someone for "Jellyfin" i.e. access to a specific server:**
+
+* **It is not us. We do not run that server.** Do not contact us, privately or publicly, for help with connecting, resetting your password, etc. We cannot help you because we do not run the server. Contact whoever you are paying. If you do not know who that is, we cannot help you.
+
+* **This is almost unequivocally piracy of some sort.** Do not discuss it with us [as per our community standards guidelines](/docs/general/community-standards).
+
+Requests for support/help with other people's servers will be ignored.
```

Comment:
Might add a sentence on reporting such server here too

Reply:
I'd rather keep that separate since this first section is for the "users" while the next section is for the "server admin".